lib/worker/main.js

```diff
@@ -1,5 +1,5 @@
 'use strict';
-const runner = require('./subprocess').getRunner();
```

Comment:
I am stuck here trying to find a way to make it work dynamically to allow a cli option as you suggested @novemberborn . Unfortunately, I don't see how this can happen without some major changes in the `lib` structure. I see many closures and static definitions, and no matter how I move pieces around, I can't manage to make it work atm.